src/component/marker/MarkerView.ts

```diff
@@ -87,6 +90,21 @@ abstract class MarkerView extends ComponentView {
             }
         });
     }
+    updateZ(ecModel: GlobalModel, markerGroupMap: HashMap<MarkerDraw>): void {
```

Comment:
Theoretically, `private` is preferable if no need to public. 

“A module should not have more access than it needs.”